README.md

```diff
@@ -84,6 +84,13 @@ tools:
 Please see [`CONTRIBUTING.md`](CONTRIBUTING.md) for more information, including
 CLAs and best practices for working with GitHub.
 
+## Built with JanusGraph
+
+* [Apache Atlas](https://github.com/apache/atlas) - metadata management for governance ([website](http://atlas.apache.org/))
+* [Eclipse Keti](https://github.com/eclipse/keti) - access control service to protect RESTful APIs ([website](https://projects.eclipse.org/projects/iot.keti))
+* [Exakat](https://github.com/exakat/exakat) - PHP static analysis ([website](https://www.exakat.io/))
+* [Windup](https://github.com/windup/windup) by Red Hat - application migration and assessment tool ([website](https://developers.redhat.com/products/rhamt/overview/))
+
```

Comment:
[Open Network Automation Platform (ONAP)](https://www.onap.org/), Linux Foundation project - automation and orchestration for Software-Defined Networks ([website](https://www.onap.org/home/community))

https://gerrit.onap.org/r/gitweb?p=aai/aai-common.git;a=blob;f=aai-core/pom.xml;h=f8761ba6448ab9ebfaa605ad8a254e9498e1de92;hb=refs/heads/beijing#l424

Reply:
Thanks! I've been looking at ONAP for a while since I saw them mention JanusGraph, but forgot to add them to this list.

I've added ONAP to my PR; I did not highlight "The Linux Foundation" as we are not highlighting the source of the other projects, and I didn't add the website link since the name already links to the same site.